[WV-953] - Wrote automated test script for checking Followers Count and Endorsers Count for topics Pro-choice and Pro-life

mythrienjam opened the pull request:
### What github.com/wevote/WebApp/issues does this fix?
[WV-953]


### Changes included this pull request?

**1) Updated OneValue.jsx**
    Added id’s for tabs “All Endorsers” and “For This Election”  [here](https://quality.wevote.us/value/pro-life)

**2) Updated topics.page.js**

**3) Added VerifyCount.js,** - Here I wrote 4 tests

     i) For checking Followers Count for Pro-choice topic
     ii) For Checking endorsers count for Pro-choice - This test should be run after the code in OneValue.jsx is merged
     iii) For checking Followers count for Pro-life 
     iv) For checking endorsers count for Pro-life  - This test should be run after the code in OneValue.jsx is merged since I have added id’s for tabs “All Endorsers” and “For This Election” tabs in  OneValue.jsx and I have used those id's in this test

 


[WV-953]: https://wevoteusa.atlassian.net/browse/WV-953?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

mythrienjam committed:
Syncing

mythrienjam committed:
Updated VerifyCount.js

charanya-QA reviewed (changes requested):
The file OneValue.jsx  already has IDs for "Forthis Election" and "Endorsers" button. Its not reflecting on the website because website is not updated with the latest code. So you can use id = "forThisElection" and  id = "allEndorsers" on the line 267 and 277 respectively.  Please change the test scripts accordingly and revert onevalue.jsx changes . Thanks!

mythrienjam commented:
Since I have latest code now on my machine,  I am able to see  id = "forThisElection" and id = "allEndorsers"  

Let me work on this complete it as soon as possible

mythrienjam committed:
Since id's were added by someone else also deleted my changes in OneValue.jsx

mythrienjam commented:
Since id's were added already , Deleted my changes in OneValue.jsx
Changed my code accordingly. 

i) Test for checking Followers Count for Pro-choice topic
 ii) Test for Checking endorsers count for Pro-choice -  It looks like ID's are not yet available on quality.wevote.us
 iii) Test for checking Followers count for Pro-life 
 iv) Test foror checking endorsers count for Pro-life  -  It looks like ID's are not yet available on quality.wevote.us

charanya-QA commented:
Thanks @mythrienjam
